pkg/k8/logs/logs.go

```diff
@@ -3,30 +3,61 @@ package logs
 import (
 	"io"
 	"os"
+	"os/signal"
+	"sync"
+	"time"
 
+	"github.com/okteto/cnd/pkg/k8/deployments"
+	appsv1 "k8s.io/api/apps/v1"
 	apiv1 "k8s.io/api/core/v1"
 	"k8s.io/client-go/kubernetes"
 	"k8s.io/client-go/rest"
 )
 
-//Logs stremas logs from a container
-func Logs(c *kubernetes.Clientset, config *rest.Config, pod *apiv1.Pod, container string) error {
-	var tailLines int64
-	tailLines = 100
-	req := c.CoreV1().Pods(pod.Namespace).GetLogs(
-		pod.Name,
-		&apiv1.PodLogOptions{
-			Container:  container,
-			Timestamps: true,
-			Follow:     true,
-			TailLines:  &tailLines,
-		},
-	)
-	readCloser, err := req.Stream()
-	if err != nil {
-		return err
+//StreamLogs stremas logs from a container
+func StreamLogs(d *appsv1.Deployment, container string, c *kubernetes.Clientset, config *rest.Config, wg *sync.WaitGroup) {
+	defer wg.Done()
+	var readCloser io.ReadCloser
+	end := false
+
+	channel := make(chan os.Signal, 1)
```

Comment:
I think it's a bit confusing to handle an os.Signal here, since it's coming from the user. I think we should have this on the `cmd` package instead, and use another channel or function call to propagate the event "down".

Reply:
Right, I will do it in a follow-up PR